source/documentation/tdb2/tdb2_admin.md

```diff
@@ -67,3 +67,13 @@ continue to be serviced.
 
 There is also a command line tool `tdb2.tdbbackup` to run the
 backup process on a database not in use.
+
```

Comment:
`tdb2.tdbbackup` has had this feature for a long time. `tdb2.tdbbackup` isn't really very important. Nowadays, tarsnaction handling applies to all databases so special casing it isn't necessary. TBH It should probably be removed.

The recent PR put the same functionality into Fuseki - they don't share code.

The Fuseki code isworks with any database type, integrates into the Fuseki lifecycle and Fuseki logging.

Could you put this change in the Fuseki docs please?

Reply:
I am a bit confused. 
You want me to drop this PR and put the same text in fuseki docs here: https://github.com/apache/jena-site/blob/main/source/documentation/fuseki2/fuseki-server-protocol.md ?